### PR TITLE
LPS-155362 Updated Input component to use onBlur, Update Types of onchange in form updated language and yup shcema

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/Form/Footer/Footer.tsx [hunk 1]
```diff
@@ -17,18 +17,25 @@ import ClayButton from '@clayui/button';
...
+}) => (
 	<ClayButton.Group spaced>
 		<ClayButton displayType="primary" onClick={onSubmit}>
-			{i18n.translate('save')}
+			{i18n.translate(primaryButtonTitle)}
 		</ClayButton>
 
 		<ClayButton displayType="secondary" onClick={() => onClose()}>
-			{i18n.translate('cancel')}
+			{isModal ? i18n.translate('close') : i18n.translate('cancel')}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/Form/Footer/Footer.tsx:38]: Simplify

```suggestion
			{i18n.translate(isModal ? 'close' : 'cancel')}
```

**Review by kevenleone — APPROVED** [review]
Awesome!

Great job, @welitonpe

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/Form/Footer/Footer.tsx (hunk 1 above)
Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/Form/Footer/Footer.tsx:38]: Only this comment about it, but you can send a follow-up PR fixing that

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/1443

**Pull request closed without merging**